src/scripts/containers/AppContainer.js

```diff
@@ -0,0 +1,9 @@
+import { connect } from "react-redux";
```

Comment:
This is the general structure of our connected `react/redux` containers. When we actually have data in our store and actions to dispatch, those will populate the functions `mapStateToProps` and `mapDispatchToProps`